docs/next_steps.md

```diff
@@ -0,0 +1,83 @@
+## Integrate Fobnail with NitroKey firmware
+
+https://github.com/Nitrokey/nitrokey-3-firmware/issues/276
+
+## Implement HID communication for Attester and Platform Owner applications
+
+This removes network stack from code base. On the other hand, it requires
+significant changes on applications running on PC, since they can no longer
+benefit from CoAP libraries and network stack built into kernel.
+
+## Add ability to provision more than one platform to Fobnail Token
+
+Currently only one platform at a time can be provisioned with one Fobnail Token.
+For easier usage for people maintaining multiple platforms, as well as for
+economic reasons, it makes sense to be able to use one Fobnail Token for
+multiple platforms.
+
+With multiple platforms supported, it should also be possible to remove a single
+platform from the list of provisioned ones. Otherwise, when one machine is
+decommissioned, all others would have to be provisioned again after clearing
+Fobnail's memory.
```

Comment:
> what for we need `qualifiedSigner`?

Qualified name uniquely identifies an object, including all its ancestor keys until the root of hierarchy (Primary Seed). By doing so, one can check if an object (e.g. key) was created on TPM under given hierarchy and not loaded from external source. I though it was part of public part of AIK but I was wrong. This actually makes sense, since in some cases the key may be loaded in different environment (not here though, because we have fixedParent and fixedTPM set). Qualified name is returned as separate structure in response to `TPM2_ReadPublic` command, perhaps we should send it to verifier during attestation as well.

> @krystian-hebel thoughts?

We could try it, but there are few drawbacks:
- Writable storage is required on host. This means that live systems and disposable VMs cannot be fully attested. Also storage drivers become part of TCB.
- Golden measurements stored on the machine are easier to obtain by the attacker.
- In the proposed approach, AIK is sent to the Token with each attestation. This is only the public part, but there were attacks that allowed to get private part from it (e.g. Coppersmith's attack), we can't be sure there won't be more.
- Attack surface is bigger - breaking either AIK or a key that is used to create AIK/RIM certificate is enough to forge the attestation results.
- SVN doesn't make single platform unprovisioning possible, you would still have to recreate certificates for platforms that aren't to be revoked. This would be easier with multiple SVNs in Fobnail memory, but then you need a way of pairing given SVN with correct certificate.